Lib/test/test_typing.py

```diff
@@ -1610,6 +1610,41 @@ class P(Protocol):
         with self.assertRaisesRegex(TypeError, "@runtime_checkable"):
             isinstance(1, P)
 
+    def test_runtime_issubclass_with_classvar_data_members(self):
+        @runtime_checkable
+        class P(Protocol):
+            x: ClassVar[int] = 1
+
+        class C: pass
+
+        class D:
+            x = 1
+        self.assertNotIsSubclass(C, P)
+        self.assertIsSubclass(D, P)
```

Comment:
One more case that is not covered: same name, different type.

```python
class F:
    x = 'a'
```

It should not pass, but maybe `ClassVar` wrapper might break / change that.